projects/plugins/jetpack/_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -2158,6 +2172,107 @@ public static function get_site_activity() {
 		);
 	}
 
+	/**
+	 * Fetch the discount for this site and return it.
+	 *
+	 * @since 10.8
+	 *
+	 * @return array|WP_Error
+	 */
+	public static function get_site_discount() {
+		$site_id = Jetpack_Options::get_option( 'id' );
+
+		if ( ! $site_id ) {
+			return new WP_Error(
+				'site_id_missing',
+				esc_html__( 'Site ID is missing.', 'jetpack' ),
+				array( 'status' => 400 )
+			);
+		}
+
+		$response = Client::wpcom_json_api_request_as_user(
+			"/sites/$site_id/discount",
+			'2',
+			array(
+				'method'  => 'GET',
+				'headers' => array(
+					'X-Forwarded-For' => ( new Visitor() )->get_ip( true ),
+				),
+			),
+			null,
+			'wpcom'
```

Comment:
Small thing, but the value being passed here (`wpcom`) is [the default value](https://github.com/Automattic/jetpack/blob/master/projects/packages/connection/src/class-client.php#L410) for the parameter, so it can probably be left off, which means the `null` argument before it can be dropped too. Unless you'd rather be extra explicit.